deploy/environments/applications.yaml

```diff
@@ -67,6 +67,7 @@ spack:
     - reportinglib%intel
     - spatial-index
     - spykfunc
+    - spykfunc@0.16.0 ^spark@3.0.0 ^py-pyspark@3.0.0
```

Comment:
should this version go into production as well? i.e. if 3.0 is only stable and we have issues with newer version, we can just provide single module for end users?

Reply:
I got stuff to work at scale with 3.1.1 as well. Seeing as it is hard to debug the interaction between Python/JVM/C++, I'd rather stay with the newer versions. Reading through the changelog, there seem to be tangential fixes to features we're using.

I've got to tune this a little, anyways.

Reply:
Ok. But you also keep `3.0` based module? `^spark@3.0.0: '{name}/{version}-spark3.0'`

Reply:
Two modules, one for 3.1 by default, one 3.0

Reply:
@matz-e / @pramodk: is the idea is to have both avaiable, in case one doesn't end up being reliable?

Reply:
Yes. From the Spark side, prior executions with 3.1 went fine, so this is really just a precautionary measure.